mail_edit/static/src/js/tour.js

```diff
@@ -0,0 +1,56 @@
+odoo.define('mail_edit.tour', function (require) {
+"use strict";
+
+var core = require('web.core');
+var tour = require('web_tour.tour');
+
+var _t = core._t;
+
+tour.register('mail_edit_tour', {
+    url: "/web",
```

Comment:
Travis is running into this:

![image](https://user-images.githubusercontent.com/1466356/107960868-72213d00-6fa5-11eb-9c36-bca7a708d19a.png)

Could it be that we have to add the `test` and `wait_for` lines? They seem to be all around in Odoo 12.0

![image](https://user-images.githubusercontent.com/1466356/107960958-911fcf00-6fa5-11eb-84b9-6b06c99e3ac4.png)